packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -72,14 +71,19 @@ export class AmplifyProjectCreator {
       process.cwd() === this.packageManagerController.projectRoot
         ? ''
         : `Navigate to your project directory using
-'cd .${this.packageManagerController.projectRoot.replace(process.cwd(), '')}'.
+'cd .${
+            this.packageManagerController.projectRoot.replace(
+              process.cwd(),
+              ''
+            ) as string
```

Comment:
why aren't the types for these values being inferred correctly?

Reply:
Good catch! Done 44d99a7af7